deploy/golden_test.go

```diff
@@ -46,21 +46,25 @@ func TestHelmRender(t *testing.T) {
     for _, f := range files {
         if !f.IsDir() && strings.HasSuffix(f.Name(), ".yaml") && !strings.HasSuffix(f.Name(), ".golden.yaml") {
             // Render this values.yaml file
-            output := helm.RenderTemplate(t,
+            output, error := helm.RenderTemplateE(t,
```

Comment:
I enhanced `golden_test.go` to also print error information so we can test for error cases too
(useful for testing schema validation etc)